docs-v2/content/en/docs/workflows/handling-platforms.md

```diff
@@ -0,0 +1,230 @@
+
+Let's test this in the same [sample Golang](https://github.com/GoogleContainerTools/skaffold/tree/main/examples/cross-platform-builds) project as before:
+
+* Run this command to build for the target architectures `linux/amd64` and `linux/arm64`:
+
+  ```cmd
+  skaffold dev -t latest --default-repo=your/container/registy --platform=linux/amd64,linux/arm64
+  ...
+  Building [skaffold-example]...
+  Target platforms: [linux/amd64,linux/arm64]
+  ...
+  [+] Building 0.3s (13/13) FINISHED
+  ...
+  => => writing image sha256:10af3142e460566f5791c48758f0040cef6932cbcb0766082dcbb0d8db7653e7
+  => => naming to gcr.io/k8s-skaffold/skaffold-example:latest_linux_amd64
+  ...
+  latest_linux_amd64: digest: sha256:15bd4f2380e99b3563f8add1aba9691e414d4cc5701363d9c74960a20fb276c4 size: 739
+  ...
+  [+] Building 52.8s (13/13) FINISHED
+  ...
+  => => writing image sha256:68866691e2d6f079b116e097ae4e67a53eaf89e825b52d6f31f2e9cc566974de
+  => => naming to gcr.io/k8s-skaffold/skaffold-example:latest_linux_arm64
+  ...
+  4e0c2525c370: Pushed
+  latest_linux_arm64: digest: sha256:868d0aec1cc7d2ed1fa1e840f38ff1aa50c3cc3d3232ea17a065618eaec4e82b size: 739
+  Build [skaffold-example] succeeded
+  ```
+
+* Validate that the image just built was multi-arch, by running the following `docker` command:
+
+  ```cmd
+  docker manifest inspect your/container/registry/skaffold-example:latest | grep -A 3 "platform"
+  ```
+
+  Outputs:
+
+  ```cmd
+    "platform": {
+        "architecture": "amd64",
+        "os": "linux"
+    }
+  --
+    "platform": {
+        "architecture": "arm64",
+        "os": "linux"
+    }
+  ```
+
+* Now if we render the Kubernetes Pod manifest for this multi-arch image, then it'll have platform affinity definition targeting both `linux/amd64` and `linux/arm64` architectures.
+
+  ```cmd
+  skaffold render --default-repo=your/container/registry --enable-platform-node-affinity
+  ```
+  
+  Outputs:
+
+  ```cmd
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: getting-started
+    namespace: default
+  spec:
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - key: kubernetes.io/os
+              operator: In
+              values:
+              - linux
+            - key: kubernetes.io/arch
+              operator: In
+              values:
+              - amd64
+          - matchExpressions:
+            - key: kubernetes.io/os
+              operator: In
+              values:
+              - linux
+            - key: kubernetes.io/arch
+              operator: In
+              values:
+              - arm64
+    containers:
+    - image: gcr.io/k8s-skaffold/skaffold-example:latest@sha256:9ecf4e52f7ff64b35deacf9d6eedc03f35d69e0b4bf3679b97ba492f4389f784
+      name: getting-started
+  ```
+
```

Comment:
nit: Does this entail all of the desired functionality here and does this reflect the currently status of the code @HEAD?  It might make sense to add a `Roadmap` section + table if not and add the current status of the code and future features (w/ possible timelines or not if that causes unnecessary strain)

Reply:
this doc represents the desired status of the feature, some things are still open in other PRs. I'd want to merge it as is since we intend to merge all these PRs before the next release.